### PR TITLE
Update DE challenge

### PR DESCRIPTION
**Pull request by kylelrichards11**
Updating the data engineering challenge to better reflect our current hiring needs.

### CODE REVIEW

File: data-engineering/README.md [hunk 1]
```diff
@@ -1,44 +1,32 @@
...
+ExampleCo, Inc is gathering several types of data for its fleet of very expensive machines. They collect four kinds of time series data for each machine in their fleet of very expensive machines. When a machine is operating in normal mode the data behaves in a fairly predictable way, but with a moderate amount of noise. Before a machine fails it will ramp into faulty mode, during which the data appears visibly quite different. Finally, when a machine fails it enters a third, and distinctly different, failed mode where all signals are very close to zero. There are three common sensors associated with each machine. There is also static data associated with each machine.
```
Comment (zachglee) [inline data-engineering/README.md:4]: `very expensive machines` is used in the first and second sentence. Was this repetition intentional?

Comment (zachglee) [reply data-engineering/README.md:4]: We say they collect 4 kinds of time series data but then later say there are 3 common sensors associated with each machine. The actual data itself appears to have 4 features. Is this a typo or something intentional?

**Review by zachglee — APPROVED** [review]
Couple minor wording questions -- other than that looks good to me! Feels clear and concise without giving too many details, so we get to see how they solve a somewhat underspecified problem!

File: data-engineering/README.md (hunk 1 above)
Comment (kylelrichards11) [reply data-engineering/README.md:4]: Good catch on the very expensive machines, I will remove the second one. And iirc from the data science challenge, one of the measurements is much noisier and has less signal than the other. But I think in this context that is not as important so I will change 3 to 4 for consistency.

File: data-engineering/README.md [hunk 2]
```diff
@@ -1,44 +1,32 @@
...
+ExampleCo, Inc is gathering several types of data for its fleet of very expensive machines. They collect four kinds of time series data for each machine in their fleet. When a machine is operating in normal mode the data behaves in a fairly predictable way, but with a moderate amount of noise. Before a machine fails it will ramp into faulty mode, during which the data appears visibly quite different. Finally, when a machine fails it enters a third, and distinctly different, failed mode where all signals are very close to zero. There are four common sensors associated with each machine. There is also static data associated with each machine.
 
-The company has contracted you to take their data extracts (linked below), map the data into your database, and summarize the results and process.
+ExampleCo stores their data in SQL tables. While SQL is useful for storing and querying data, it is less useful as input for machine learning pipelines. At Tagup, our ML Pipelines expect data as an n-dimensional array, and a common first step is converting a customer's data from SQL to array form.
```
Comment (irvinktang) [inline data-engineering/README.md:6]: personally, when seeing "array form", it's pretty clear to me that it is referring to the aforementioned n-dimensional array. so i think the wording here is good the way it is

**Pull request merged**